lib/addons/external-secrets/index.ts

```diff
@@ -23,7 +23,7 @@ const defaultProps: ExternalsSecretsAddOnProps = {
   name: "external-secrets",
   chart: "external-secrets",
   release: "blueprints-addon-external-secrets",
-  version: "0.10.4",
+  version: "0.16.1",
```

Comment:
based on this page: https://external-secrets.io/latest/introduction/stability-support/

external-secrets 0.16.1 only support K8S: 1.32.  

does it means blueprint also drop support for k8s: 1.32?

also we saw in the release page, there is some promotion need to be done. Those setups are handled by blueprints or not?